### DISCUSSION TITLE
Resetting EMQX Frontend/Web UI user password via console using Portainer

### DISCUSSION THREAD

**Discussion by talon-k** [hassio-addons/addon-emqx#95] [General]
_Removed content, as it is not supported by the Home Assistant project and is considered bad practice._

**Comment by frenck** [comment]
The use of portainer (and Docker directly that way) isn't supported by the Home Assistant project. As a matter of fact, installing Portainer will lead to an unsupported installation.

Therefore, I'm strong **discourage** the above and recommend no-one to use it.

../Frenck

**Reply by talon-k** [reply]
What would be the correct way of resetting a user password in that case? I'm all for doing it the correct way, so to speak.
